plugins/kotlin/idea/tests/test/org/jetbrains/kotlin/idea/editor/quickDoc/PackageQuickDocTest.kt

```diff
@@ -0,0 +1,29 @@
+// Copyright 2000-2023 JetBrains s.r.o. and contributors. Use of this source code is governed by the Apache 2.0 license.
+package org.jetbrains.kotlin.idea.editor.quickDoc
+
+import com.intellij.codeInsight.javadoc.JavaDocInfoGenerator
+import com.intellij.psi.PsiPackage
+import org.jetbrains.kotlin.idea.test.KotlinLightCodeInsightFixtureTestCase
+import org.junit.Assert
+
+class PackageQuickDocTest: KotlinLightCodeInsightFixtureTestCase() {
+    fun testNoDuplicates() {
+        val psiFile = myFixture.addFileToProject(
+            "com/example/nodupes/Fantastic.kt", """
+            package com.example.nod<caret>upes
+            class Fantastic
+        """.trimIndent()
+        )
+        myFixture.configureFromExistingVirtualFile(psiFile.virtualFile)
```

Comment:
I think it would be nice to make it more generic and use `testData` as many other tests do.

Reply:
I don't understand how that makes it more generic. This test is small and contained and minimizes external dependencies. It's clear what's being tested directly from the test.

I assume those tests use `testData` because it would be unwieldy to set up complex file structures/contents within a unit test. But this test only needs to add a single, empty class.